src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,41 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setName("Audi");
+        manufacturer.setCountry("Germany");
+        manufacturerService.create(manufacturer);
+        manufacturer.setName("Peugeot");
+        manufacturer.setCountry("France");
+        manufacturer = manufacturerService.create(manufacturer);
```

Comment:
You are creating the same manufacturer twice? This is not a good practice. Better, create two separate variables, for example, `audi` and `peugeot`, and save them.